beetsplug/rewrite.py

```diff
@@ -60,7 +58,7 @@ def __init__(self):
                 raise ui.UserError("invalid field name (%s) in rewriter" %
                                    fieldname)
             self._log.debug(u'adding template field {0}', key)
-            pattern = re.compile(pattern.lower())
+            pattern = re.compile(pattern)
```

Comment:
This makes the pattern case-sensitive, yes? If so, can we please restore case-insensitivity so existing configurations continue to work?